__tests__/app.test.js

```diff
@@ -157,8 +156,77 @@ describe("GET /api/articles/:article_id/comments", () => {
         .get('/api/articles/50/comments')
         .expect(404)
         .then(({ body }) => {
-            expect(body.msg).toBe("No comments found for that id")
+            expect(body.msg).toBe("No article found with that id")
+        })
+    })
+
+    it("should respond with 200 and empty array if id is valid but has no comments", () => {
+        return request(app)
+        .get('/api/articles/2/comments')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.articleComments).toEqual([])
         })
     })
+})
 
+describe("POST /api/articles/:article_id/comments" , () => {
+    it("should accept comment object and respond with 201 and posted comment" , () => {
+        const testId = 2
+        const testComment  = {
+            username: "butter_bridge",
+            body: "wow what an epic comment"
```

Comment:
An epic comment indeed! Would expect nothing less from the great butter_bridge 😎